src/software/ai/hl/stp/play/free_kick_play_test.cpp

```diff
@@ -37,7 +37,7 @@ TEST_F(FreeKickPlayTest, test_free_kick_play_on_enemy_half)
         [](std::shared_ptr<World> world_ptr, ValidationCoroutine::push_type& yield) {
             while (world_ptr->getMostRecentTimestamp() < Timestamp::fromSeconds(9.5))
             {
-                yield();
+                yield("Timestamp not yet 9.5s");
```

Comment:
```suggestion
                yield("Timestamp not at 9.5s");
```
To match other error messages